.env.prodlike

```
@@ -3,5 +3,40 @@
 # Real deployments should use a secret store, not this file.
 
 ANALYTICS_HASH_SECRET=prodlike-local-secret-not-for-production
-DATABASE_URL=http://127.0.0.1:9000
-NEXT_PUBLIC_SITE_URL=http://127.0.0.1:3001
+# Local SQL Server defaults for prod-like validation.
+# Override these in .env.prodlike.local or via shell env if needed.
+DB_HOST=127.0.0.1
+DB_PORT=1433
+DB_NAME=kravhantering
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PORT key

(UnorderedKey)